.gitignore

```diff
@@ -211,4 +211,5 @@ FakesAssemblies/
 GeneratedArtifacts/
 _Pvt_Extensions/
 ModelManifest.xml
-/Master/NucleusCoop.VC.VC.opendb
+**/*.VC.VC.opendb
```

Comment:
The pattern "**/*.VC.VC.opendb" is the same as "*.VC.VC.opendb". To be honest, I didn't realize that, when I made the commit, but I knew that "**/*.VC.VC.opendb" meant anywhere, including the current directory.